site/docs/spark-queries.md

```diff
@@ -143,18 +146,17 @@ To show table history, run:
 ```sql
 SELECT * FROM prod.db.table.history
 ```
-```text
-+-------------------------+---------------------+---------------------+---------------------+
-| made_current_at         | snapshot_id         | parent_id           | is_current_ancestor |
-+-------------------------+---------------------+---------------------+---------------------+
+
+<div class="markdown-table-container" markdown="block">
+| made_current_at | snapshot_id  | parent_id | is_current_ancestor |
+| -- | -- | -- | -- |
 | 2019-02-08 03:29:51.215 | 5781947118336215154 | NULL                | true                |
 | 2019-02-08 03:47:55.948 | 5179299526185056830 | 5781947118336215154 | true                |
 | 2019-02-09 16:24:30.13  | 296410040247533544  | 5179299526185056830 | false               |
 | 2019-02-09 16:32:47.336 | 2999875608062437330 | 5179299526185056830 | true                |
 | 2019-02-09 19:42:03.919 | 8924558786060583479 | 2999875608062437330 | true                |
 | 2019-02-09 19:49:16.343 | 6536733823181975045 | 8924558786060583479 | true                |
-+-------------------------+---------------------+---------------------+---------------------+
-```
+</div>
```

Comment:
@szehon-ho here, and I use the css test in my local branch
```css
.markdown-table-container {
  width: 780px;
  overflow-x: scroll;
}
```
![image](https://user-images.githubusercontent.com/20125927/141053950-e8c22b13-d1a4-45a4-bb50-770043e5e7e3.png)

Reply:
Thanks for pointing this out @KnightChess ! If we're going to use the container class for all markdown tables, we can use `overflow-x: auto;` instead and it will only show the bar when there's overflow.

Reply:
To prevent a merge conflict, I'll mention this over in PR #3422 so the css can be updated there.